test/plugins/builds.test.js

```diff
@@ -740,7 +740,7 @@ describe('build plugin test', () => {
     });
 
     describe('GET /builds/{id}/secrets', () => {
-        const id = 'd398fb192747c9a0124e9e5b4e6e8e841cf8c71c';
+        const id = '12345';
```

Comment:
There are some peculiarities I don't fully understand about whether or not hapijs is going to parse `{id}` in the path to a number or leave it a string. Since the test is using `withArgs(id)`, I felt it best to have the id be whatever type hapi passes in to the model. The model doesn't really care if it gets a string or a number as it will convert the input to a number anyways.